CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.9.0] - 2020-03-09
```

Comment:
There were no breaking changes so according to semver bumping patch is sufficient for version 0.

```suggestion
## [0.8.1] - 2020-03-09
```